lisa/features/disks.py

```diff
@@ -74,10 +74,11 @@ def get_resource_disk_mount_point(self) -> str:
         raise NotImplementedError
 
     def get_resource_disks(self) -> List[str]:
-        return []
+        raise NotImplementedError
 
     def get_resource_disk_type(self) -> schema.ResourceDiskType:
-        return schema.ResourceDiskType.SCSI
+        controller_type = self.get_os_disk_controller_type()
```

Comment:
The method is called `get_resource_disk_type`, but it calls `get_os_disk...`. It's inconsistent between name and logic.

Reply:
Fixed.